content/doc/developer/FIPS-140/index.adoc

```diff
@@ -0,0 +1,47 @@
+---
+layout: developerchapter
+summary: FIPS-140 for developers
+wip: true
```

Comment:
Optional.  I've not found the "work in progress" flag to be very helpful for developer documentation.  They are all a work in progress.  This page has good information and useful information.  The reader doesn't gain much by marking it a work in progress, though I'm also fine if you choose to leave it as a work in progress.
```suggestion
```

Reply:
sure.  I think it is important that to convey that the list is not complete.

perhaps a specific admonition instead?

Reply:
No problem to leave it as work in progress.  That's a good way of saying that more information is coming in the future.